src/app/layout.tsx

```diff
@@ -62,6 +62,10 @@ const RootLayout: React.FC<RootLayoutProps> = ({ children }) => {
         </AppRouterCacheProvider>
         <Analytics />
         <SpeedInsights />
+        <script
```

Comment:
I'm seeing a few issues that need to be considered:
1. The chat widget crashes the site when the settings are changed in the [admin console ](https://my.small.chat/channel/-Nr6BQKab7S5oGEyXeei/behavior)
2. Unable to reliably whitelist/blacklist routes in the console. The ideal path would be to use the [API](https://docs.small.chat/javascriptapi) but it requires a paid subscription. Another solution is to write a custom solution using a flag in the routes.ts file on applicable urls. 

I'm punting on this feature for now.

Reply:
I feel good about allowing on all pages and controlling with a feature flag in case the feature doesn't behave.